Detections/W3CIISLog/ProxyShellPwn2Own.yaml

```diff
@@ -0,0 +1,42 @@
+id: 968358d6-6af8-49bb-aaa4-187b3067fb95
+name: Exchange ProxyShell Pwn2Own
+description: |
+  'This query looks for suspicious request patterns to Exchange servers that fit patterns recently
+  blogged about by PeterJson.
+  Reference: https://peterjson.medium.com/reproducing-the-proxyshell-pwn2own-exploit-49743a4ea9a1'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureMonitor(IIS)
+    dataTypes:
+      - W3CIISLog
+queryFrequency: 1d
+queryPeriod: 14d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+query: |
+  let successCodes = dynamic([200, 302]);
```

Comment:
I also saw 401 status for SSRF into “/powershell endpoint stage in the blog. 
Should we include that to this status list ?

Reply:
I've added that status code in. In all honesty the detection is tight enough we could remove the status codes, however we'd be unable to filter out attempts vs. success. I'll use 200, 302 and 401 for now but if we start seeing lots of status code diversity we can consider removing this check.